cmake/kokkos_tpls.cmake

```
@@ -88,3 +88,9 @@ IF (KOKKOS_ENABLE_MEMKIND)
    SET(KOKKOS_ENABLE_HBWSPACE)
    LIST(APPEND KOKKOS_MEMSPACE_LIST HBWSpace)
 ENDIF()
+
+KOKKOS_OPTION(ENABLE_COMPILE_TIME_PERF OFF BOOL "Enable per-file compile time measurements")
```

I think you're supposed to do
```suggestion
KOKKOS_ENABLE_OPTION(COMPILE_TIME_PERF OFF "Enable per-file compile time measurements")
```